docs/serving/knative-kubernetes-services.md

```diff
@@ -33,41 +44,40 @@ networking-istio         1         1         1            1           1h
 webhook                  1         1         1            1           1h
 ```
 
-## Service: activator
+These serivces and deployments are installed by the `serving.yaml` file during install. The next section describes the function of these components.
 
-The responsibilities of the activator are:
+## Components
 
-- Receiving & buffering requests for inactive Revisions.
-- Reporting metrics to the autoscaler.
-- Retrying requests to a Revision after the autoscaler scales such Revision
-  based on the reported metrics.
+### Service: activator
 
-## Service: autoscaler
+The activator is responsible for receiving & buffering requests for inactive revisions and reporting metrics to the autoscaler. In addition, it retries requests to a revision after the autoscaler scales the revision based on the reported metrics.
 
-The autoscaler receives request metrics and adjusts the number of pods required
-to handle the load of traffic.
+### Service: autoscaler
 
-## Service: controller
+The autoscaler receives request metrics and adjusts the number of pods required to handle the load of traffic.
+
+### Service: controller
 
 The controller service reconciles all the public knative objects and autoscaling
-CRDs. When a user applies a knative service to the kubernetes api, this creates
-the config and route. It will convert config into revisions. It will convert
-Revision into Deployment and KPA.
+CRDs. When a user applies a Knative service to the Kubernetes API, this creates
+the configuration and route. It will convert the configuration into revisions, and the revisions into Deployment and Knative Pod Autoscaling (KPA).
+
+### Service: webhook
 
-## Service: webhook
+The webhook intercepts all Kubernetes API calls as well as all CRD insertions and updates.
+It sets default values, rejects inconsitent and invalid objects, and validates and mutates Kubernetes API calls.
 
-The webhook intercepts all kubernetes api calls, all crd insertions and updates.
-It does two things:
+### Deployment: networking-certmanager
 
-1. Set default values
-2. Rejects inconsitent and invalid objects.
+The certmanager reconciles cluster ingresses into cert manager objects.
 
-It validates and mutates k8s api calls.
+### Deployment: networking-istio
 
-## Deployment: networking-certmanager
+The networking-istio deployment reconciles a cluster's ingress into a virtual service.
 
-The certmanager reconciles cluster ingress into cert manager objects.
+## What's Next
 
-## Deployment: networking-istio
+1. For a deeper look at the services and deployments involved in Knative Serving, click [here](https://github.com/knative/serving/blob/master/docs/spec/overview.md#service).
```

Comment:
Nice! Love that you added this section. I'd vote for making this an unordered list (just bullets), though.